tests/TestRunner/app/ApiTests.js

```diff
@@ -556,7 +558,7 @@ describe(module.id, function () {
         expect(result).toBe('method called');
     });
 
-    if (TNSIsConfigurationDebug) {
+    if (false && TNSIsConfigurationDebug) {
```

Comment:
The false && part was to disable these lengthy tests while developing, we should remove it.